### PR TITLE
Replace missing block for outside of sphere surface

### PR DESCRIPTION
**Pull request by pshriwise**
This (re)adds support for the outer halfspace of sphere surfaces along with a new test to ensure it's working correctly.

### CODE REVIEW

File: test/test_local.py [hunk 1]
```diff
@@ -34,6 +34,21 @@ def test_planes(request, run_in_tmpdir):
...
 
 
+@reset_openmc_ids
+def test_nested_spheres(request, run_in_tmpdir):
+    inner_sphere = openmc.Sphere(r=10.0)
+    middle_sphere = openmc.Sphere(r=20.0)
+    outer_sphere = openmc.Sphere(r=30.0)
+
+    inner_cell = openmc.Cell(region=-inner_sphere)
+    middle_cell = openmc.Cell(region=-middle_sphere)
```
Comment (eepeterson) [inline test/test_local.py:44]:
```suggestion
    middle_cell = openmc.Cell(region=+inner_sphere & -middle_sphere)
```
Shouldn't these be nested?

File: test/test_local.py [hunk 2]
```diff
@@ -34,6 +34,21 @@ def test_planes(request, run_in_tmpdir):
...
+    outer_cell = openmc.Cell(region=-outer_sphere)
```
Comment (eepeterson) [inline test/test_local.py:45]:
```suggestion
    outer_cell = openmc.Cell(region=+middle_sphere & -outer_sphere)
```
same thing here

File: test/gold/nested_spheres.jou
Comment (eepeterson) [inline test/gold/nested_spheres.jou:1]: I don't see any subtract commands here since I don't think the test case was defined properly

File: test/test_local.py [hunk 3]
```diff
@@ -34,6 +34,21 @@ def test_planes(request, run_in_tmpdir):
...
+
+    g = openmc.Geometry([outer_cell, middle_cell, inner_cell])
```
Comment (eepeterson) [inline test/test_local.py:47]: Is the order that the cells are listed in going to have an impact? I was expecting to see the outputs in the cubit journal file ordered by cell ID, but it looks reversed.

**Review by eepeterson — CHANGES_REQUESTED** [review]
Thanks for the quick fix on this @pshriwise but I think there are a couple things that need tweaking.

File: test/test_local.py (hunk 3 above)
Comment (eepeterson) [reply test/test_local.py:47]: could also be worth tacking on a vacuum boundary condition to the outer sphere so that it makes a viable geometry that could ostensibly be run in openmc

Comment (pshriwise) [reply test/test_local.py:47]: > Is the order that the cells are listed in going to have an impact? I was expecting to see the outputs in the cubit journal file ordered by cell ID, but it looks reversed.

I believe they'll be generated in the order they appear in the iterable passed to the Geometry object. Corresponding IDs on the volume objects in the resulting CAD would be great, but might be difficult to manage based on how Cubit handles ID spaces. An easier short-term solution would be to place each cell in it's own group with the ID in the name for reference to the original model.

> could also be worth tacking on a vacuum boundary condition to the outer sphere so that it makes a viable geometry that could ostensibly be run in openmc

Agreed that it would be nice and I'm happy to include those changes here. Right now I'm planning on spending my time covering and verifying more geometry capabilities  in the test suite (hopefully correctly in the future) than every test case being runnable in OpenMC.

Comment (eepeterson) [reply test/test_local.py:47]: super thanks!

**Review by eepeterson — APPROVED** [review]

**Review by eepeterson — APPROVED** [review]
thanks for the fix @pshriwise!
